velox/type/TimestampUtil.h

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include "velox/common/base/Exceptions.h"
+
+namespace facebook::velox {
+
+struct TimestampToStringOptions {
+  enum Precision : int8_t {
+    kMilliseconds = 3,
+    kNanoseconds = 9,
+  } precision = kNanoseconds;
+
+  bool zeroPaddingYear = false;
+  char dateTimeSeparator = 'T';
+
+  enum class Mode : int8_t {
+    kFull,
+    kDateOnly,
+    kTimeOnly,
+  };
+
+  Mode mode = Mode::kFull;
+};
+
+/// A static class that holds helper functions for TIMESTAMP type.
+class TimestampUtil {
```

Comment:
I'm not a fan of XxxUtil classes as they tend to become a dumping ground for all sorts of unrelated functionality. Wondering, why not add epochToUtc and tmToString as static method to Timestamp class?

Reply:
I think it should be ok to put `epochToUtc` and `tmToString` as static methods in the `Timestamp` class. so I'll move it back.